src/js/utils/ConfirmModal.jsx

```diff
@@ -13,6 +13,7 @@ const ConfirmModal = ({
   onClose,
   buttons,
   className,
+  hideCloseButton = false,
```

Comment:
nitpicky, but we used to set default prop values in the `Component.defaultProps`

Reply:
I can also see this prop missing in the `propTypes`

Reply:
yup, forgot about this one